src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+                        //                                2. The actual deadline after "/by " till end of string
+                        tasks[itemIndex] = new Deadline((command.substring(DEADLINE_SIZE, command.indexOf("/"))),
+                                dateOrTime);
+                    }
+                    // Create tasks for class Event
+                    else if (command.contains("event")) {
+                        // Same substring read as Deadline
+                        tasks[itemIndex] = new Event((command.substring(EVENT_SIZE, command.indexOf("/"))),
+                                dateOrTime);
+                    }
+                    // If no keywords are used, stores new tasks into Task array
+                    else {
+                        tasks[itemIndex] = new Task(command);
+                    }
+                }
+                itemIndex++;
+                System.out.println("Will do sir, I've added: " + System.lineSeparator() + "  "
+                        + tasks[itemIndex - 1].printTask());
+                if (itemIndex == 1) {
+                    System.out.printf("Now you have %d task in your list.\n", itemIndex);
+                } else {
+                    System.out.printf("Now you have %d tasks in your list.\n", itemIndex);
+                }
+            }
+        }
+    }
+
+    public static void printWelcome() {
+        String logo = "        _|       _|_|        _|_|_|      _|      _|    _|_|_|        _|_|_|\n"
+                + "        _|     _|    _|      _|    _|    _|      _|      _|        _|\n"
+                + "        _|     _|_|_|_|      _|_|_|      _|      _|      _|          _|_|\n"
+                + " _|     _|     _|    _|      _|    _|      _|  _|        _|              _|\n"
+                + "  _| _|    _|  _|    _|  _|  _|    _|  _|    _|    _|  _|_|_|  _|  _|_|_|\n";
+        System.out.print("Initialising...............\n");
+        System.out.println(LINE + System.lineSeparator() + logo);
+        // Prints enclosed text in italics
+        String byline = "\033[3m----------------------------------Just a rather very intelligent system---------\033[0m\n";
+        System.out.println(byline + "Good Evening Sir! I'm J.A.R.V.I.S");
+        System.out.println("How may I be of assistance to you today?");
+        System.out.println(LINE + System.lineSeparator());
+    }
+
+    public static void printBye() {
+        System.out.println("Affirmative sir, I'll shut down all operations");
+    }
+
+    public static void printList(Task[] tasks, int itemIndex) {
+        System.out.println(LINE + System.lineSeparator() + "Here are the current tasks in your list:");
+        for (int count = 0; count < itemIndex; count++) {
+            System.out.println(count + 1 + "." + tasks[count].printTask());
+        }
+        System.out.println(LINE);
+    }
+
+    public static void doneMethod(Task[] tasks, int itemIndex, String command) {
```

Comment:
Consider changing the method name to `setTaskAsDone`, so that it will be more descriptive to the reader